Collibri/Repositories/DbImplementation/DbSectionRepository.cs

```diff
@@ -0,0 +1,66 @@
+using Collibri.Models;
+using Collibri.Repositories.ExtensionMethods;
+
+namespace Collibri.Repositories.DbImplementation;
+
+public class DbSectionRepository : ISectionRepository
+{
+    private readonly DataContext _context;
+        
+    public DbSectionRepository(DataContext dataContext)
+    {
+        _context = dataContext;
+    }
+    
+    public Section? CreateSection(Section section)
+    {
+        if (_context.Sections.Any(sections => sections.Equals(section)))
```

Comment:
cia reikia _context.Sections.ToList(),Any del custom equal, arba pakeist, kad nebutu to metodo isvis sekcijoj ir butu ta salyga linq